app/src/test/kotlin/com/intive/picover/common/converters/BitmapConverterTest.kt

```diff
@@ -0,0 +1,66 @@
+package com.intive.picover.common.converters
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.ImageDecoder
+import android.net.Uri
+import android.provider.MediaStore
+import com.intive.picover.common.sdk.AndroidSdkVersion
+import io.kotest.core.spec.style.ShouldSpec
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkObject
+import io.mockk.mockkStatic
+import io.mockk.unmockkAll
+import io.mockk.verify
+
+class BitmapConverterTest : ShouldSpec(
```

Comment:
None of the tests is checking the value returned after calling `convertBitmapToBytes`

Reply:
Yes, you have a point I should add this test

Reply:
Done 😄